types/adhan/index.d.ts

```diff
@@ -0,0 +1,93 @@
+// Type definitions for adhan-js 3.0
+// Project: https://github.com/batoulapps/adhan-js
+// Definitions by: Ikramullah <https://github.com/skymunn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Start declare module
+// declare module 'adhan' { export = Adhan }

```

Comment:
Did you mean to export this here?

[adhan](https://github.com/batoulapps/adhan-js#node) looks like a UMD library

https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#umd

Reply:
Yes, Adhan is UMD library. I can call it globally when using <script> in HTML.